.travis.yml

```diff
@@ -2,18 +2,35 @@ language: ruby
 rvm:
   - 2.4.3
 
+before_install:
+  - openssl aes-256-cbc -K $encrypted_a3d58fa41b45_key -iv $encrypted_a3d58fa41b45_iv -in scripts/id_rsa.enc -out scripts/id_rsa -d
```

Comment:
@patcon Can you explain how this works and what are `id_rsa.enc` and `encrypted_a3d58fa41b45_key`?

Reply:
Is $encrypted_a3d58fa41b45_key a Travis CI env var?

Reply:
It's what travis walks you through the setup of, after using the `travis encrypt-file` command. I'll link the docs for clarity :)